code/datums/gameticker.dm

```diff
@@ -766,7 +766,19 @@ var/global/current_state = GAME_STATE_WORLD_INIT
 
 	//logTheThing("debug", null, null, "Zamujasa: [world.timeofday] finished spacebux updates")
 
-
+	var/list/playtimes = list() //associative list with the format list("ckeys\[[player_ckey]]" = playtime_in_seconds)
```

Comment:
Note that this stuff won't happen on manual restarts of the server or other such stuff. But those are rare so it doesn't matter much. Could always be moved into `Reboot` later.

Also I'd prefer this to be in a separate proc but that's just a detail really.